proposals/infra/INF-0002-root-signature-in-clang.md

```diff
@@ -493,8 +494,50 @@ TODO
   In DXIL generation, values like ShaderVisibility/ParameterType need to be
   checked to make sure they are in correct range.
 
-##### StaticSampler
-- Comparison filter must have ComparisonFunc not equal to 0.
+- ShaderVisibility
+  - SHADER_VISIBILITY_ALL
```

Comment:
SHADER_VISIBLITY_ALL is listed twice here.

Reply:
Fixed.